<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouterUVWindow</class>
 <widget class="QDialog" name="ajouterUVWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UTProfiler - Ajouter une nouvelle UV</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="layoutCode">
      <item>
       <widget class="QLabel" name="lblCode">
        <property name="text">
         <string>Code de la nouvelle UV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtCodeNewUV"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutDescription">
      <item>
       <widget class="QLabel" name="lblDescription">
        <property name="text">
         <string>Description de la nouvelle UV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtDescNewUV"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutCredits">
      <item>
       <widget class="QLabel" name="lblCredits">
        <property name="text">
         <string>Cr√©dits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableCredits"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutSemestres">
      <item>
       <widget class="QCheckBox" name="chkAutomne">
        <property name="text">
         <string>Automne</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkPrintemps">
        <property name="text">
         <string>Printemps</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutBtns">
      <item>
       <widget class="QPushButton" name="btnAccepter">
        <property name="text">
         <string>Ajouter la nouvelle UV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAnnuler">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
